extensions/data-transfer/portability-data-transfer-imgur/src/main/java/org/datatransferproject/datatransfer/imgur/photos/ImgurPhotosImporter.java

```diff
@@ -52,20 +52,24 @@
   private final ObjectMapper objectMapper;
   private final TemporaryPerJobDataStore jobStore;
   private final Monitor monitor;
-  private static final String BASE_URL = ImgurTransferExtension.BASE_URL;
-  private static final String CREATE_ALBUM_URL = BASE_URL + "/album";
-  private static final String UPLOAD_PHOTO_URL = BASE_URL + "/image";
+
+  private static String createAlbumUrl;
```

Comment:
Likewise, if we can remove static and make both of these final.

Reply:
Done